org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/services/LanguageClient.java

```diff
@@ -141,4 +143,23 @@
 	default void semanticHighlighting(SemanticHighlightingParams params) {
 		throw new UnsupportedOperationException();
 	}
+
+	/**
+	 * This request is sent from the server to the client to ask the client to create a work done progress.
+	 */
+	@JsonRequest("window/workDoneProgress/create")
+	default CompletableFuture<Void> createProgress(WorkDoneProgressCreateParams params) {
+		throw new UnsupportedOperationException();
+	}
+	
+	/**
+	 * The base protocol offers also support to report progress in a generic fashion. 
+	 * This mechanism can be used to report any kind of progress including work done progress 
+	 * (usually used to report progress in the user interface using a progress bar) 
+	 * and partial result progress to support streaming of results.
+	 */
+	@JsonRequest("$/progress")
+	default CompletableFuture<Void> notifyProgress(ProgressParams params) {
```

Comment:
This should remain a `@JsonNotification`, with `void` return type:
https://microsoft.github.io/language-server-protocol/specifications/specification-current/#progress

My comment about `@JsonRequest` was related only to `window/workDoneProgress/create`.